build/syntax-highlight.ts

```diff
@@ -46,14 +46,13 @@ const loadAllLanguages = lazy(() => {
     "regex",
     "rust",
     "scss",
-    "svelte",
     "sql",
     "toml",
     "tsx",
     "typescript",
     "uri",
     "wasm",
-    // "webidl", // block list
+    "web-idl",
```

Comment:
This works as expected, adding per your suggestion:

```suggestion
    "webidl",
```